libraries/RainMaker/examples/RMakerSwitch/RMakerSwitch.ino

```diff
@@ -94,6 +94,8 @@ void setup() {
 
   RMaker.enableSystemService(SYSTEM_SERV_FLAGS_ALL, 2, 2, 2);
 
+  WiFiProv.initProvision(NETWORK_PROV_SCHEME_BLE, NETWORK_PROV_SCHEME_HANDLER_FREE_BTDM);
```

Comment:
There should be a #if CONFIG_IDF_TARGET_ESP32S2 to use a SOFTAP scheme like is done in `WiFiProv.beginProvision`, instead of forcing the BLE.
```
#if CONFIG_IDF_TARGET_ESP32S2
  WiFiProv.beginProvision(NETWORK_PROV_SCHEME_SOFTAP, NETWORK_PROV_SCHEME_HANDLER_NONE, NETWORK_PROV_SECURITY_1, pop, service_name);
#else
  WiFiProv.beginProvision(NETWORK_PROV_SCHEME_BLE, NETWORK_PROV_SCHEME_HANDLER_FREE_BTDM, NETWORK_PROV_SECURITY_1, pop, service_name);
#endif
```

Reply:
yes, pushed the chnages.